src/ext/processObliv/processObliv.ml

```diff
@@ -49,6 +49,9 @@ let oblivIntTarget = ref dummyType
 let oblivShortTarget = ref dummyType
 let oblivLongTarget = ref dummyType
 let oblivLLongTarget = ref dummyType
+let oblivFloatTarget = ref dummyType (*CHANGE*)
```

Comment:
Apologies! The original purpose of those markers was for me to keep track of changes across commits. I thought I removed all of them but clearly I overlooked a few. I can definitely fix this for the PR.